Models/Certificate.swift

```diff
@@ -175,8 +187,16 @@ private enum MethodsForV1_1 {
                 return nil
         }
         
+        var issuedOnDate : Date?
+        
+        if let iod = dateFormatter.date(from: issuedOnString) as Date? {
+            issuedOnDate = iod
+        } else {
+            issuedOnDate = dateFormatter2.date(from: issuedOnString)
+        }
+        
         let signatureImage = imageData(from: signatureImageURI)
-        return Assertion(issuedOn: issuedOnDate,
+        return Assertion(issuedOn: issuedOnDate!,
```

Comment:
If the `issuedOnString` doesn't match either format, this method will blow up here. You should probably add a `guard` check to make sure that `issuedOnDate` isn't nil after line 196